examples/twittermap/web/public/javascripts/common/services.js

```diff
@@ -40,13 +39,14 @@ angular.module('cloudberry.common', [])
       }
     };
   })
-  .service('cloudberry', function($http, $timeout, $location, cloudberryConfig) {
+  .service('cloudberry', function($http, $timeout, $location, cloudberryConfig, ResultCache) {
```

Comment:
Name suggestion: MapResultCache
$http, $timeout, $location is needed? Please check. If not, let's remove them.

Reply:
$http and $location have been removed, since they are not used.